README.md

```diff
@@ -243,9 +243,33 @@ HTTP(port).Register("/penny", [](Request r) {
   }
 });
 ```
-HTTP server also has support for several other features, check out the [`bricks/net/api/test.cc`](https://github.com/KnowSheet/Bricks/blob/master/net/api/test.cc) unit test.
+```cpp
+// Returning a potentially indefinite response chunk by chunk.
+HTTP(port).Register("/chunked", [](Request r) {
+  const size_t n = atoi(r.url.query["n"].c_str());
+  const size_t delay_ms = atoi(r.url.query["delay_ms"].c_str());
+    
+  const auto sleep = [&delay_ms]() {
+    std::this_thread::sleep_for(std::chrono::milliseconds(delay_ms));
+  };
+    
+  auto response = r.SendChunkedResponse();
+
+  sleep();
+  for (size_t i = 0; n && i < n; ++i) {
+    response(".");  // Use double quotes for a string, not single quotes for a char.
+    sleep();
+  }
+  response("\n");
+});
 
-**TODO(dkorolev)**: Chunked response example, with a note that it goes to Sherlock.
+EXPECT_EQ(".....\n", HTTP(GET("test.tailproduce.org/chunked?n=5&delay_ms=2")).body);
```

Comment:
Then it would a) require a port, and b) invite a non-savvy developer, who just `git clone`-d this repo to try copy-pasting the code and assume it would work.

We can make `test.tailproduce.org` work for them later on (and, probably, we will). But not their localhost :)